data/wild/flee_mons.asm

```diff
@@ -31,6 +31,5 @@ OftenFleeMons: ; 3c5a8
 AlwaysFleeMons: ; 3c5b1
 	db RAIKOU
 	db ENTEI
-;	db SUICUNE
```

Comment:
why remove suicune here?

Reply:
All the other suicune comments had already been removed, since Suicune simply does not roam in Crystal. Just finishing the job here.